conf/services/online-payments.yml

```diff
@@ -1,5 +1,5 @@
 serviceKey: online-payments
-serviceName: Online Payments
+serviceName: Online Payments Service
```

Comment:
With this change we end up with 'Service service' in the text 'This page only contains information about the Online Payments Service service, available' We could append 'Service' in the <h1> title.

Reply:
@matthewmascord Thanks for the feedback, have amended the messages and tests.